.gitignore

```diff
@@ -348,3 +348,6 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+# Exclude appsettings.json files to prevent the changes of connection strings
+appsettings.json
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** Please return the appsettings.json because otherwise the services will complain if we were to pull the repository.‏